src/java/org/apache/cassandra/service/accord/AccordJournalTable.java

```
@@ -149,6 +154,38 @@ public RangeSearcher rangeSearcher()
         return new TableRangeSearcher();
     }
 
+    public void start()
+    {
+        if (index == null) return;
+        Index tableIndex = cfs.indexManager.getIndexByName(AccordKeyspace.JOURNAL_INDEX_NAME);
+        RetryStrategy retry = DatabaseDescriptor.getAccord().retry_journal_index_ready.retry();
```

Are there scenarios under which we do not want to wait for journals readiness? Maybe we should just log where we are stuck but I'd say we should always wait for it.

---

> but I'd say we should always wait for it.

partial startup puts us in a bad state, so rather fail startup or block waiting rather than log and move on... the reason we are doing this patch is that the "move on" part ends up causing issues =D